src/Twig/IconExtension.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Tabler bundle, created by Kevin Papst (www.kevinpapst.de).
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace KevinPapst\TablerBundle\Twig;
+
+use Symfony\UX\Icons\IconRenderer;
+use Twig\Extension\RuntimeExtensionInterface;
+
+final class IconExtension implements RuntimeExtensionInterface
+{
+    /**
+     * @param array<string, string> $icons
+     */
+    public function __construct(
+        private readonly IconRenderer $iconRenderer,
+        private readonly array $icons
+    ) {
+    }
+
+    public function createIcon(string $name, bool $withIconClass = false, string $default = null): string
+    {
+        $icon = $this->getIcon($name, $default);
+
+        if (str_contains($icon, ':')) {
+            return $this->iconRenderer->renderIcon($icon, ['class' => 'icon']);
+        }
+
+        @trigger_error('Support for webfonts is deprecated. Switch to UX icons. Using twig function tabler_icon("' . $name . '") is deprecated.', \E_USER_DEPRECATED);
```

Comment:
Using twig function tabler_icon("' . $name . '") without an iconify identifier is deprecated